src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/FAA/FaaVacancyStatusSummary.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Esfa.Recruit.Vacancies.Client.Infrastructure.Services.FAA
+{
+    // Duplicate of das-vacancyservicesshared SFA.Apprenticeships.Core/Entities/Application/Applications/VacancyStatusSummary.cs
+    // Ensure you keep both classes in sync.
+    public class FaaVacancyStatusSummary
+    {
+        // Note: Type is `int` in FAA but `long` here, to comply with ID length in this solution
+        //   FAA needs to change - recorded as tech-debt in V2
+        public long LegacyVacancyId { get; set; }
```

Comment:
I'd change this to int and handle when we new up the message. This class needs to be exact replica of FAA so at least the messages we can create are guaranteed to be deserialized by FAA.

Reply:
@chrisjensenuk 
If we approve 1 vacancy per second, 24 hours a day, every day of the week, this will take over 50 years to overflow.